RevenueCatUI/CustomerCenter/ManageSubscriptionsButtonStyle.swift

```diff
@@ -33,7 +33,7 @@ struct ManageSubscriptionsButtonStyle: PrimitiveButtonStyle {
         let textColor = color(from: appearance.buttonTextColor, for: colorScheme)
 
         Button(action: { configuration.trigger() }, label: {
-            configuration.label.frame(width: 300)
+            configuration.label.frame(maxWidth: .infinity)
```

Comment:
👍

Reply:
I was struggling with this, as a full width card didn't look good with the text size I had. Looks much better with the larger text! 🙌